birdhouse/README.rst

```diff
@@ -0,0 +1,259 @@
+.. role:: raw-html-m2r(raw)
+   :format: html
+
+
+Docker instructions
+-------------------
+
+Requirements:
+
+
+* 
+  Centos 7 or Ubuntu Bionic (18.04), other distros untested.
+
+* 
+  Hostname of Docker host must exist on the network.  Must use bridge
+  networking if Docker host is a Virtual Machine.
+
+* 
+  User running ``pavics-compose.sh`` below must not be ``root`` but a regular user
+  belonging to the ``docker`` group.
+
+* 
+  Install latest docker-ce and docker-compose for the chosen distro (not the
+  version from the distro).
+
+To run ``docker-compose`` for PAVICS, the `\ ``pavics-compose.sh`` <pavics-compose.sh>`_ wrapper script must be used.
```

Comment:
Links to `pavics-compose.sh`, `env.local.example`, and `scripts/create-wps-pgsql-databases.sh`, `scripts/create-magpie-users`, `scripts/bootstrap-instance-for-testsuite`, `vagrant_variables.yml.example`, `vagrant-utils/disk-resize` broken when browsing on github.